Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
Just to explain, registers in processors don't really have the C "rules".  For example, an "s8" on the PSP's processor is:

```
+--------------------------------+
+                        SIIIIIII+
+--------------------------------+
                         ^ sign bit
                          ^ integer bits
 ^ other bits that are ignored
```

In a way, they're always u32 values.  The "s8" rule just explains which portion of those bits are relevant, and what they mean.

Here, we're taking a register value (`regval`), and using `'i'` to indicate "this register which contains 32 bits of data is to be understood as an s32".  There are other cases (other than `'i'`), where it's treated differently.

As far as signed/int, this code you're changing is actually adhering to an ABI.  It's IMHO less clear to use `signed` when we're certain `regval` will always be a `u32`.  If this code were to be reused for a 64-bit MIPS emulator (like the PS2) it would need significant changes, anyway.

-[Unknown]